flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobManagerSharedServicesTest.java

```diff
@@ -111,7 +112,7 @@ private JobManagerSharedServices buildJobManagerSharedServices(Configuration con
             throws Exception {
         return JobManagerSharedServices.fromConfiguration(
                 configuration,
-                new BlobServer(configuration, new VoidBlobStore()),
+                new BlobServer(configuration, new File("foobar"), new VoidBlobStore()),
```

Comment:
shouldn't this use a proper tmp directory?

Reply:
Yes, I think so. Will update it.